reports/relatorio-google-trends.Rmd

```diff
@@ -0,0 +1,335 @@
+
+```
+
+
+```{r, message=FALSE}
+meio_ambiente_apelido <- read_csv("../data/temas/meio_ambiente/pl_do_veneno_apelido.csv") %>% 
+  mutate(tema = "Meio ambiente")
+
+agenda_nacional_apelido <- read_csv("../data/temas/agenda_nacional/reforma_tributaria_apelido.csv") %>% 
+  mutate(tema = "Agenda nacional")
+
+```
+
+```{r, message=FALSE}
+comparacao_meio_ambiente <- read_csv("../data/temas/meio_ambiente/comparacao_pl_do_veneno.csv")
+comparacao_agenda_nacional <- read_csv("../data/temas/agenda_nacional/comparacao_reforma_tributaria.csv")
+
+```
+
+```{r, message=FALSE}
+formata_para_visualizacao_semana_anterior_semana_posterior <- function(df_apelido) {
+  semana_apelido <- df_apelido %>% 
+  mutate(semana = epiweek(date)) %>% 
+  group_by(apelido, semana) %>% 
+  summarise(pop_semana = max(pop))
+
+semana_apelido %>% 
+  arrange(desc(semana)) %>% 
+  head(2) %>% 
+  arrange(apelido, semana) %>% 
+  mutate(pop_ant = lag(pop_semana)) %>% 
+  arrange(desc(semana))
+}
+
+create_grafico_semana_anterior_semana_posterior <- function(data, prop_maximo) {
+  ggplot(data=data,aes(x=apelido, y=pop_semana))+
+  geom_bar(aes(y=100),stat="identity",position ="identity",alpha=0.4,fill='grey', width = 0.4) +
+  geom_bar(aes(y=pop_ant), stat="identity", position ="identity", alpha=1, fill='pink', width = 0.3) +
+  geom_bar(aes(y=pop_semana), stat="identity", position ="identity", alpha=.5, fill='red', width = 0.2) +
+  coord_cartesian(ylim = c(0, 100)) +
+  xlab("Apelido") + 
+  ylab("Pressão") +
+  ggplot2::ggtitle("Pressão Recente") +
+  scale_y_continuous(labels = function(x) paste0(x, "%"))+
+  theme_minimal() +
+  annotate("text", x=1, 
+           y= 95, label = format(as.Date(prop_maximo$date), "%d/%m/%Y"))
+}
+```
+
+Nesta análise estavamos interessados na popularidade de algumas proposições. Utilizamos, então, o Google Trends — ferramenta do Google que fornece dados sobre pesquisa de termos no buscador — para investigar o interesse das pessoas nessas proposições. Traduzimos esse interesse para 'Pressão Externa' ou só 'Pressão', utilizando-a como amostra da pressão exercida pela população.
+
+##Pressão Recente
+
+Abaixo podemos visualizar a pressão atual e da semana anterior para duas proposições.
+
+A Pressão da semana atual é a vermelha escura e a pressão da semana anterior é a cor de rosa mais larga (a largura das barras serve exclusivamente para indicar a sobreposição), enquanto a maior pressão alcançada é representada pela barra cinza.
+
+###PEC 06/2019 - Nova Previdência
+
+```{r fig.align='center'}
+
+atual_popularidade_previdencia <- formata_para_visualizacao_semana_anterior_semana_posterior(previdencia_apelido)
+
+prop_maximo <- previdencia_apelido %>% 
+  filter(pop == 100)
+
+p <- create_grafico_semana_anterior_semana_posterior(atual_popularidade_previdencia, prop_maximo)
+ggplotly(p)
+```
+
+Como podemos perceber, a pressão externa da Reforma da Previdência subiu na última semana.
```

Comment:
É bom colocar as semanas específicas para as quais a análise foi realizada, porque última semana depende do referencial. Assim, fica mais fácil de verificar.

Reply:
Além disso, seria bom procurar nas notícias possíveis causas para o aumento da pressão e elencar aqui.

Reply:
As notícias que encontrei na semana em questão aconteceram após a criação do relatório.